test/evergreen.yml

```diff
@@ -6044,16 +6057,7 @@ buildvariants:
   # We run on medium as small has too small a disk.
   - ubuntu2004-perf-testing
   expansions:
-    configure_env_vars: PATH=/opt/mongodbtoolchain/v4/bin:$PATH
```

Comment:
This looks like a pretty standard expansions setting (after simplification by this PR). Should we consider making some expansions template to further reduce duplication? (again, happy it to be covered in a separate PR/ticket)